sktime/markov/msm/markov_state_model.py

```diff
@@ -225,7 +225,11 @@ def stationary_distribution(self):
     def update_stationary_distribution(self, value: np.ndarray):
         r""" Explicitly sets the stationary distribution, re-normalizes """
         if value is not None:
-            self._stationary_distribution = np.copy(value) / np.sum(value)
+            sum = np.sum(value)
+            if sum != 0.:
```

Comment:
out of curiosity do we ever expect this case?

Reply:
it _should_ not happen but it did happen during one of my tests so i thought i'd add the check

Reply:
how do you feel about raising a warning or something?

Reply:
actually ill roll back the check, numpy itself spits out a warning if a division by 0 is attempted